packages/server/src/api/controllers/migrations.ts

```diff
@@ -27,7 +27,7 @@ export async function getMigrationStatus(ctx: Ctx) {
 
   const latestAppliedMigration = await getAppMigrationVersion(appId)
 
-  const migrated = latestAppliedMigration === getLatestMigrationId()
+  const migrated = latestAppliedMigration === getLatestEnabledMigrationId()
```

Comment:
Does this need to instead be a `>` check? Thinking about the situation where a migration is dynamically disabled after having been applied (never intended to happen but we _are_ creating the environment for it to happen if mistakes are made). If that does happen, we still want to consider the app migrated if the latest migration is _greater than_ the latest enabled migration.